alf/utils/dist_utils.py

```diff
@@ -434,6 +430,37 @@ def greedy_fn(dist):
         return nest.map_structure(greedy_fn, nested_distributions)
 
 
+def get_mode(dist):
```

Comment:
Need to comment that it may not be the actual mode for TransformedDistribution

Reply:
Comments added following suggestion.